charts/evm-rollup/values.yaml

```diff
@@ -10,7 +10,7 @@ images:
   geth:
     repo: ghcr.io/astriaorg/astria-geth
     tag: 0.13.0
-    devTag: latest
+    devTag: pr-38
```

Comment:
Just double-checking -- is this tag for all of development intentional?

Reply:
Was intentional since this PR is dependent on `astria-geth` [#38](https://github.com/astriaorg/astria-geth/pull/38). @joroshiba suggested waiting until this PR merges, updating geth PR with latest protos and merging, then making a monorepo cleanup PR to update `devTag` to `latest`.

Reply:
Yeah, because we have proto changes here that needs geth update, it gets a little weird here:
1. get this pr submitted which pushes prod proto changes
2. update pr-38 of geth to utilize new main proto libraries, update and submit.
3. update mono repo back to latest.

A bit of chicken/egg problem here. My preference is to keep astria-geth main from relying on non-main protos. We should wait to submit PR here until pr-38 is reviewed and approved though so we don't end up out of sync.